app/services/requests/user/get-user-request.js

```diff
@@ -0,0 +1,27 @@
+// app/services/requests/user/get-user-request.js
+import Service, { inject as service } from '@ember/service';
+import RequestManager from '@ember-data/request';
+import Fetch from '@ember-data/request/fetch';
+
+export default class RequestsUserGetUserRequestService extends Service {
```

Comment:
That is already better than previous attempt. What about just `UsersService` ? `Request` is imo just implementation detail we don't have to show it in the name. 
Then I imagine we can have clear methods with equivalents API:

`User.getUsers()` <--already available
`User.getUser(:id)`
`User.create(:attrs)`
`User.query(:queryParamsForSearch)`

etc

Reply:
I understand - makes sense - so just have something like - 'users-service' that controls all the request stuff in included methods. I had been thinking of having the requests split into atomic files, but perhaps that overkill here - like:
```
get-users-request.js
create-user-request.js
delete-user-request.js
etc..
```

But one file with all those methods I suppose is unlikely to grow or get too complicated given all it does.

Reply:
Changed here: https://github.com/Mr-Smyth/task-manager/pull/17/commits/f2fd2326a3a1356127074befebe6c482ea0c274d

Reply:
Yeah, we can take a more atomic approach, if we observe that our service has too many responsibilities. CRUD for starting point, seems like a middle ground :ok_hand: